### DISCUSSION TITLE
Failing check for both CLI and Gradle on 1.19.0 for issues in the baseline.

### DISCUSSION THREAD

**Discussion by richdapice** [detekt/detekt#4532] [Q&A]
I recently upgraded to 1.19.0 and updated my rules as well, although I already had `SpacingBetweenPackageAndImports` and `MaxLineLength` and they were both active.

I am able to generate a baseline file after deleting the previous baseline I see it being created. The problem is, after generating the baseline I run my check but I fail for issues that ARE IN the baseline.

```
SpacingBetweenPackageAndImports - [HttpClient.kt] - Log

<ID>SpacingBetweenPackageAndImports:HttpClient.kt$ </ID> - baseline.xml
```

I have checked my detekt configuration multiple times and wasn't having this issue before the update.
I am failing the check locally and as a github action.

Thanks!

**Comment by BraisGabin** [comment]
This issue was introduced by #4335 and it should be fixed by #4504

**Reply by marschwar** [reply]
The problem seems very similar but #4504 will not solve it.
